base/.mysql_profile

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+if [[ -z $MYSQL_HOST ]]; then
+  if [[ -z $STORAGE_PORT_3306_TCP_ADDR ]]; then  
+    echo "** ERROR: You need to link with a MySQL container as 'storage' or specify MYSQL_HOST env var."
+    echo "STORAGE_PORT_3306_TCP_ADDR (container link) or MYSQL_HOST should contain a MySQL hostname."
+    exit 1
+  fi
+  MYSQL_HOST=$STORAGE_PORT_3306_TCP_ADDR
+fi
+
+export MYSQL_HOST
```

Comment:
These are the minimum variables needed to link the containers. anybody can add more variables with docker environment. The main reason for adding USER/PASS is to prevent people from having to know the values the mysql image bakes in. makes sense?